target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -187,6 +187,10 @@ elecom,wrh-300cr)
 	set_wifi_led "$boardname:green:wlan"
 	ucidef_set_led_netdev "lan" "lan" "$boardname:green:ethernet" "eth0"
 	;;
+engenius,esr600)
+	ucidef_set_led_netdev "wlan5g" "5.0GHz" "$boardname:blue:wlan5g" "wlan0"
```

Comment:
I'd personally use just "5GHz" there instead of "5.0GHz". I think the latter is used rarely. But it's a matter of taste.

Reply:
I would have done that too, but the case label for that LED is "5.0GHz", so I went with that.

Reply:
Okay, fine.